src/mgr.sol

```diff
@@ -95,18 +95,18 @@ contract TinlakeManager {
     // Events
     event Rely(address usr);
     event Deny(address usr);
```

Comment:
Conversation got resolved but the `indexed` is still missing for `Rely` and `Deny`

Reply:
fixed